kernel-commits.mk

```diff
@@ -2,8 +2,12 @@ KERNEL_COMMIT_amd64_v5.10.186_generic = d61682724485
 KERNEL_COMMIT_amd64_v6.1.38_generic = 8859f43ee8c9
 KERNEL_COMMIT_amd64_v6.1.38_rt = 37bc37278441
 KERNEL_COMMIT_amd64_v6.1.68_generic = 45e0ac394428
+KERNEL_COMMIT_amd64_v6.1.105_rt = 8fe1d58eff31
```

Comment:
I'm just curious: why is 6.1.106 ok for generic but not for RT, which requires 6.1.105 =)

Reply:
there is no stable v6.1.106 for RT available